server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/routes/JMAPAPIRoutes.scala

```diff
@@ -0,0 +1,86 @@
+package org.apache.james.jmap.routes
+
+import java.io.InputStream
+import java.util.stream
+import java.util.stream.Stream
+
+import eu.timepit.refined.auto._
+import io.netty.handler.codec.http.HttpHeaderNames.CONTENT_TYPE
+import io.netty.handler.codec.http.HttpMethod
+import io.netty.handler.codec.http.HttpResponseStatus.OK
+import org.apache.james.jmap.HttpConstants.JSON_CONTENT_TYPE
+import org.apache.james.jmap.JMAPUrls.JMAP
+import org.apache.james.jmap.json.Serializer
+import org.apache.james.jmap.method.CoreEcho
+import org.apache.james.jmap.model.Invocation.{Arguments, MethodName}
+import org.apache.james.jmap.model.{Invocation, RequestObject, ResponseObject}
+import org.apache.james.jmap.{Endpoint, JMAPRoute, JMAPRoutes}
+import org.slf4j.{Logger, LoggerFactory}
+import play.api.libs.json.{JsError, JsSuccess, Json}
+import reactor.core.publisher.Mono
+import reactor.core.scala.publisher.SMono
+import reactor.core.scheduler.Schedulers
+import reactor.netty.http.server.{HttpServerRequest, HttpServerResponse}
+
+class JMAPAPIRoutes extends JMAPRoutes {
+  override def logger(): Logger = LoggerFactory.getLogger(getClass)
+
+  private val coreEcho = new CoreEcho
```

Comment:
We should rely on guice to inject a set of Method.

Reply:
maybe not for now

Reply:
Ok let's keep it for later then